universal-login-commons/lib/utils/changingMessageFields.ts

```diff
@@ -1,5 +1,5 @@
 import {utils} from 'ethers';
-import {SignedMessage} from '@universal-login/commons';
+import {SignedMessage} from '../.';
```

Comment:
Typo?

Reply:
File was moved to commons. It works, but i can change this for '../'